### PR TITLE
[14.0][MIG] account_cutoff_start_end_dates

### PR DESCRIPTION
**Pull request by alexis-via**
This module account_cutoff_start_end_dates is a merge of account_cutoff_prepaid (from v13) and account_cutoff_accrual_dates (from v12, because it seems nobody ported it to v13), as discussed on issue #133

### CODE REVIEW

**Comment by fclementic2c** [comment]
@alexis-via 
It works perfectly. I love this option : "Partner on Move Line" - really great that it is included in the module now.

I have improvement suggestions - if you agree : 
I would like to see the accounting date on the cutoff lines (see the printscreen below). I think it is easier when you audit / double check numbers:

screenshot-3477014-153-220b27 runbot1-3 odoo-community org-2021 04 20-09_21_01

**Review by fclementic2c — APPROVED** [review]

**Review by omarie-ctc — APPROVED** [review]
It works well for me

File: account_cutoff_start_end_dates/README.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+   :local:
+
+Configuration
+=============
+
+On the form view of the company, in the *Configuration* tab, there is a *Cut-offs* section where you can configure:
+
+* the *Default Cut-off Journal*, which is the journal in which the cut-off journal items will be generated by default,
+* the *Default Account for Prepaid Revenue*,
+* the *Default Account for Prepaid Expense*.
```
Comment (fredzamoabg) [inline account_cutoff_start_end_dates/README.rst]: I can not find the Configuration tab definition with Cut-offs section in company view but just in **Invoicing > Configuration > Settings** as described in **account_cutoff_base** module

File: account_cutoff_start_end_dates/models/account_cutoff.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+    start_date = fields.Date(readonly=True)
+    end_date = fields.Date(readonly=True)
+    total_days = fields.Integer(readonly=True)
+    cutoff_days = fields.Integer(
+        readonly=True,
+        help="In regular mode, this is the number of days after the "
+        "cut-off date. In forecast mode, this is the number of days "
+        "between the start date and the end date.",
+    )  # Old name: prepaid_days
```
Comment (fredzamoabg) [inline account_cutoff_start_end_dates/models/account_cutoff.py:224]: did you plan to crate a migration script for this?

Comment (fredzamoabg) [inline account_cutoff_start_end_dates/models/account_cutoff.py:224]: based https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0, field name changes should be follow of a migration script, did you plan to crate any?

**Review by fredzamoabg — CHANGES_REQUESTED** [review]

File: account_cutoff_start_end_dates/models/account_cutoff.py (hunk 2 above)
Comment (fredzamoabg) [reply account_cutoff_start_end_dates/models/account_cutoff.py:224]: @alexis-via any news about that?

Comment (mmequignon) [reply account_cutoff_start_end_dates/models/account_cutoff.py:224]: @alexis-via Is there a migration script planned?

Comment (fredzamoabg) [reply account_cutoff_start_end_dates/models/account_cutoff.py:224]: @mmequignon see https://github.com/akretion/account-closing/pull/4

**Comment by alexis-via** [comment]
@fredzamoabg Thanks for your contribution on the migration scripts!

**Comment by simahawk** [comment]
@alexis-via it's all :red_circle: Could you check?

**Comment by simahawk** [comment]
@alexis-via tnx!
@fredzamoabg can you update your review?

File: account_cutoff_start_end_dates/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2016-2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2018-2021 CampToCamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Cut-off Prepaid",
```
Comment (TDu) [inline account_cutoff_start_end_dates/__manifest__.py]:
```suggestion
    "name": "Account Cut-off Start End Dates",
```

File: account_cutoff_start_end_dates/README.rst (hunk 1 above)
Comment (TDu) [reply account_cutoff_start_end_dates/README.rst]: Indeed. All configuration has been moved into Settings under the Accounting section within a Cut-offs sub section, which is a good thing. Unfortunately the documentation has not followed.

**Review by TDu — APPROVED** [review]
Thank you for adding the migration script :+1:

File: account_cutoff_start_end_dates/models/account_cutoff.py (hunk 2 above)
Comment (TDu) [reply account_cutoff_start_end_dates/models/account_cutoff.py:224]: Migration scripts have been integrated

**Pull request merged**